cmd/eksctl-anywhere/cmd/list.go

```diff
@@ -0,0 +1,15 @@
+package cmd
```

Comment:
Added this to get the jobs passing, but will rebase after Rajashree adds it in her PR.